modules/gitops-argo-cd-installation.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * argo-cd-custom-resource-properties.adoc
+
+:_content-type: PROCEDURE
+[id="gitops-argo-cd-installation_{context}"]
+= Installing Argo CD 
+
+[role="_abstract"]
+To manage cluster configurations or deploy applications, you can install and deploy a new Argo CD instance. To install the Argo CD instance follow the steps given below:
+
+.Procedure
+
+. Create or select the project where you want to install the Argo CD instance. 
```

Comment:
I don't know the product, so I can't tell. 
- If it is a lead-in sentence and adds clarity, how about:
 ```suggestion
Create or select the project where you want to install the Argo CD instance: 
```
- Otherwise, if it is a distinct step, you might need to provide additional guidance on where or how to get there.